setup.py

```diff
@@ -1,7 +1,7 @@
 import ast
 import os
 from io import open
-
+from cve_bin_tool.version import VERSION
```

Comment:
I believe this doesn't work correctly when you do it in setup.py because during a new package install, setup.py is run before cve_bin_tool is installed, so cve_bin_tool.version fails (or gets the version associated with the previous install).  that's why it's got the file read instead.

If you scrap the setup.py changes, though, the changes for doc/conf.py should be all we need to fix this issue anyhow.